iphone/CoreApi/CoreApi/PlacePageData/Common/PlacePagePreviewData.mm

```diff
@@ -46,6 +50,16 @@ static PlacePageDataSchedule convertOpeningHours(std::string_view rawOH)
 
 @implementation PlacePagePreviewData
 
+- (instancetype)initWithTrackInfo:(TrackInfo * _Nonnull)trackInfo {
+  self = [super init];
+  if (self) {
+    // TODO: (KK) Replace separator with a shared static constant.
```

Comment:
nit: This todo won't improve anything in terms of speed/reducing allocations and can be safely removed.